core/libesphttpd_base64.h

```diff
@@ -0,0 +1,6 @@
+#ifndef LIBESPHTTPD_BASE64_H
```

Comment:
can you replace this with #pragma once ?

Reply:
Of course, I was just following the existing style ;)

Reply:
@PaulFreund I've been moving that way since its a much simpler approach. Most all compilers support it.